apps/tai/test/tai/venue_adapters/bitmex/stream/process_auth/messages/update_account_test.exs

```diff
@@ -77,4 +77,25 @@ defmodule Tai.VenueAdapters.Bitmex.Stream.ProcessAuth.Messages.UpdateAccountTest
       assert updated_account.locked == Decimal.new("1.11558098")
     end
   end
+
+  describe "with no margin data" do
+    test "skips account update" do
```

Comment:
This test should use ` refute_receive {:account_store, :after_put, _}`.

How do you feel about also extracting a `setup_account` helper and using it in each of these tests? e.g.

```elixir
defp setup_account do
   {:ok, _} =
        Tai.Venues.Account
        |> struct(venue_id: @venue, credential_id: @credential_id, asset: :btc, type: "default")
        |> Tai.Venues.AccountStore.put()
end
```

Reply:
I'm bypassing GenServer in this case because the pattern matching for that specific response message needs to work. `refute_receive` does not test the failure because the process crashes after `cast` is called.